packages/core/src/utilities/validation.js

```diff
@@ -314,9 +314,9 @@ export const validators = {
 
 export const hasValue = (value: Value): boolean => {
   const valueIsEmptyArray = Array.isArray(value) && value.length === 0;
-  const hasNoValue =
```

Comment:
I've add some tests below for this changes

Reply:
Thanks for adding these tests!